src/baseCode/index.js

```diff
@@ -59,4 +59,4 @@ governing permissions and limitations under the License.
       window[instanceName].q = [];
     }
   });
-})(window, ["alloy"]);
+};
```

Comment:
One of the consequences of doing this is that the code emitted to the console during our build process, which I believe is intended to be copied into the installation instructions within our user docs, is different/larger than the base code we already have in the instructions. For example, here's what we have currently:

```
!function(n,o){o.forEach(function(o){n[o]||((n.__alloyNS=n.__alloyNS||
[]).push(o),n[o]=function(){var u=arguments;return new Promise(
function(i,l){n[o].q.push([i,l,u])})},n[o].q=[])})}
(window,["alloy"]);
```

And here's what's now emitted to the console during the build process:

```
!function(){"use strict";var n;(function(n){n.forEach((function(n){window[n]||(window.__alloyNS=window.__alloyNS||
[],window.__alloyNS.push(n),window[n]=function(){var o=arguments;return new Promise(
(function(w,i){window[n].q.push([w,i,o])}))},window[n].q=[])}))})
(["alloy"])}();
```

It seems we would rather use the old one than the new one. I'm not sure how to best solve this problem. We might be able to maintain this file as it previously was, and then do some magic during build time to manipulate the file content into a module. Another option would be to maintain two separate files with similar content (both the old file and this new module-based one). Anyway, it's something to think about.

Reply:
I modified the rollup & terser options and code so that the same output is generated.